Assets/Scripts/Player/bob.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class bob : MonoBehaviour
+{
+    Vector3 startPos;
+    public float amplitude = 10f;
+    public float period = 1f;
+    private Transform body;
+
+    protected void Start()
```

Comment:
FPS games should not have a bob as the character moves, it throws off the aim of the player